crates/sui-core/src/authority.rs

```diff
@@ -1101,14 +1101,12 @@ impl AuthorityState {
             execution_engine::manual_execute_move_call_fake_txn_digest(sender, move_call.clone());
         let temporary_store =
             TemporaryStore::new(self.database.clone(), input_objects, transaction_digest);
-        let storage_gas_price = self
-            .database
-            .get_sui_system_state_object()?
-            .parameters
-            .storage_gas_price
-            .into();
-        let gas_status =
-            SuiGasStatus::new_with_budget(MAX_TX_GAS, storage_gas_price, storage_gas_price);
+        let gas_status = SuiGasStatus::new_with_budget(
+            MAX_TX_GAS,
+            // TODO: Use proper computation gas price.
```

Comment:
Note that in dev_inspect_move_call, we seem to be using the storage price as computation gas price. Not sure if that's a big deal.
@tnowacki @amnn

Reply:
I believe @tnowacki changed this in #7372, but to a case where the unit of computation is always 1.